stable/redis/templates/metrics-deployment.yaml

```diff
@@ -53,10 +53,12 @@ spec:
         {{- end }}
         env:
         - name: REDIS_ADDR
-        {{- if .Values.cluster.enabled }}
+        {{- if and .Values.cluster.enabled .Values.sentinel.enabled }}
+          value: {{ printf "%s:%d" ( include "redis.fullname" . ) ( int .Values.sentinel.service.redisPort ) | quote }}
```

Comment:
Just for more context, why don't we want to check the information from the redis processes as well?

Reply:
I tried configuring `REDIS_ADDR` to use both ports (redisPort and sentinelPort) and only redisPort. The result is very similar:

```console
$ kubectl port-forward svc/redis-metrics 9121:9121 &
$ kubectl port-forward svc/redis2-metrics 9122:9121 &
$ curl -Lo metrics-one-port.txt 127.0.0.1:9121/metrics
$ curl -Lo metrics-two-ports.txt 127.0.0.1:9122/metrics
$ cat metrics-one-port.txt | wc
     375    1452   21578
$ cat metrics-two-ports.txt | wc
     375    1449   21961
```

They metrics exported are exactly the same.